src/modules/vault/states/useVaultState.ts

```diff
@@ -9,23 +9,21 @@ const setIsVisibleBalance = (isVisible: 'true' | 'false') =>
   localStorage.setItem(IS_VISIBLE_KEY, isVisible);
 
 interface State {
```

Comment:
remove this file, and control status and data by file `src/modules/vault/hooks/details/useVaultDetails.ts`

Reply:
Done